### PR TITLE
Add Kannel sms provider documentation

### PR DESCRIPTION
**Pull request by Ayaan49**
Added Kannel sms provider documentation as per issue #116

### CODE REVIEW

File: channels-and-providers/sms/kannel.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    
+    ```
+    Make sure to replace /path/to/kannel.conf with the actual path to your Kannel configuration file.
+
+# Creating a Kannel integration with Novu
+
+  After setting up your Kannel follow these steps to integrate with novu:
+
+ 1. Visit the [Integrations Store](https://web.novu.co/integrations) on Novu.
+ 2. Locate **Kannel** and click on the `Disabled` button and mark it as `Active`.
```
Comment (unicodeveloper) [inline channels-and-providers/sms/kannel.mdx:56]: You omitted the Add a provider option. Please check a recent guide https://github.com/novuhq/docs/blob/staging/channels-and-providers/email/maqsam.mdx#L17 . The step 2 and 3 needs to be similar.

File: channels-and-providers/sms/kannel.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    ```bash
+
+    bearerbox /path/to/kannel.conf
+    
+    ```
+    Make sure to replace /path/to/kannel.conf with the actual path to your Kannel configuration file.
+
+# Creating a Kannel integration with Novu
+
+  After setting up your Kannel follow these steps to integrate with novu:
```
Comment (unicodeveloper) [inline channels-and-providers/sms/kannel.mdx:53]: there should be a comma after Kannel

**Review by unicodeveloper — COMMENTED** [review]
Please attend to the comments

**Comment by Ayaan49** [comment]
@unicodeveloper I have rectified the errors. Is it okay?

**Review by unicodeveloper — APPROVED** [review]

**Comment by unicodeveloper** [comment]
We greatly appreciate your contribution. We hope you'll stay with us and continue to be an integral part of this project.

Please consider giving this repo a star and joining our community discord server, if you haven't already, to access more resources and connect with like-minded individuals. [Discord Community here 🎉 ](https://discord.gg/novu) #Hacktoberfest

**Pull request merged**